recipes/openamundsen/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "openamundsen" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openamundsen-{{ version }}.tar.gz
+  sha256: cfcc305352be6cacb0c07f923b2b646f25481e4a86da595d021e25631e13fca9
+
+build:
+  number: 0
+  skip: true   # [py<37]
```

Since this is pure Python better to make this noarch:
```suggestion
  noarch: python
```